Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -618,7 +618,7 @@ def addKalmanTracks(
         inputInitialTrackParameters="estimatedparameters",
         outputTrajectories="trajectories",
         directNavigation=directNavigation,
-        pickTrack=-1,
+        pickTrack=0,
```

Comment:
what's the difference between `0` and `-1`?

Reply:
this is a leftover